--- Makefile.orig	2007-10-04 12:07:36.000000000 +0800
+++ Makefile	2007-10-04 12:08:38.000000000 +0800
@@ -14,13 +14,13 @@
 
 install:
 	$(INSTALL) -d -m 755 $(bindir)
-	$(INSTALL) -m 755 -o 0 abcde $(bindir)
-	$(INSTALL) -m 755 -o 0 cddb-tool $(bindir)
+	$(INSTALL) -m 755 abcde $(bindir)
+	$(INSTALL) -m 755 cddb-tool $(bindir)
 	$(INSTALL) -d -m 755 $(mandir)
-	$(INSTALL) -m 644 -o 0 abcde.1 $(mandir)
-	$(INSTALL) -m 644 -o 0 cddb-tool.1 $(mandir)
+	$(INSTALL) -m 644 abcde.1 $(mandir)
+	$(INSTALL) -m 644 cddb-tool.1 $(mandir)
 	$(INSTALL) -d -m 755 $(etcdir)
-	$(INSTALL) -m 644 -o 0 abcde.conf $(etcdir)
+	$(INSTALL) -m 644 abcde.conf $(etcdir)
 
 tarball:
 	@cd .. && tar czvf abcde_$(VERSION).orig.tar.gz \
