--- abcde-2.9.3/Makefile.FIX_MAK	2019-02-07 15:21:37.735646896 +0200
+++ abcde-2.9.3/Makefile	2019-02-07 15:23:49.814404009 +0200
@@ -1,14 +1,14 @@
 abcde_version = abcde-2.9.3
 INSTALL = /usr/bin/install -c
 
-prefix = /usr/local
-exec_prefix = $(prefix)
+prefix =
+exec_prefix = $(prefix)/usr
 bindir = $(exec_prefix)/bin
 sysconfdir = /etc
-datarootdir = $(prefix)/share
-docdir = $(datarootdir)/doc/$(abcde_version)
+datarootdir = $(prefix)/usr/share
+docdir = $(datarootdir)/doc/abcde
 mandir = $(datarootdir)/man
-DESTDIR =
+# DESTDIR =
 
 all:
 
